esx_service/vmci/vmci_server.c

```diff
@@ -154,6 +154,7 @@ vmci_get_one_op(const int s,    // socket to listen on
                ret, strerror(errno), b, MAGIC);
       close(client_socket);
       errno = saved_errno;
+      CHECK_ERRNO(ret);
```

Comment:
Redundant since errno is set on earlier line. If it is for sake of consistency then you want to add CHECK_ERRORNO(ret) at other places in file too for e.g. L 169, L176